user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,133 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+Below you can find detailed information on how you can register a private Catalog item, either [using the Catalog UI](#using-the-catalog-ui), or [using the Catalog API](#using-the-catalog-api).
+
+## Using the Catalog UI
+
+If you use the UI, you can only register a new Catalog item, but not a new version. To register a new version for an existing item, [use the API](#using-the-catalog-api).
+
+To register a new Catalog item:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon in the top-right corner.
+
+1. Click the *Register item* button.
+
+1. Select the organization for which you want to register the Catalog item.
+
+   The selection box is automatically populated with all the organizations you have access to on the dataminer.services platform.
+
+1. Select the type of the Catalog item.
+
+   This can be one of the supported [Catalog item types](xref:About_the_Catalog_module#supported-catalog-item-types). The type will define how the Catalog item will be shown to the users. Users can also filter on type to quickly find what they are looking for when browsing the Catalog.
```

Comment:
If only the supported catalog item types can be selected, I'd rephrase this as follows to prevent possible confusion:
```suggestion
   The selected [Catalog item type](xref:About_the_Catalog_module#supported-catalog-item-types) defines how the Catalog item will be shown to the users. Users can also filter on type to quickly find what they are looking for when browsing the Catalog.
```